src/Shared/Resources/Strings.shared.resx

```diff
@@ -278,10 +278,15 @@
     <value>MSB5028: Solution filter file at "{0}" includes project "{1}" that is not in the solution file at "{2}".</value>
     <comment>{StrBegin="MSB5028: "}UE: The project filename is provided separately to loggers.</comment>
   </data>
+  <data name="WildcardResultsInDriveEnumeration" xml:space="preserve">
+    <value>MSB5029: The value "{0}" of the "{1}" attribute in element &lt;{2}&gt; is an invalid wildcard that results in drive enumeration.</value>
```

Comment:
Should this mention undefined properties?

Reply:
It could suggest that this could occur due to an undefined property, but that may not always be the case -

Ex: 
```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <Target Name="CreateItemTest">
        <CreateItem Include="\*.log" Exclude="\**\*.log"/>
    </Target>
 </Project>
```